### PR TITLE
Install test,fmt,clippy CI jobs

### PR DESCRIPTION
**Pull request by connorkuehl**

### CODE REVIEW

**Review by squidboylan — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
```
Comment (nordzilla) [inline .github/workflows/test.yml]: > nitpick

Please add a newline at the end of file.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          components: clippy
+      - uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: -- -D warnings
```
Comment (nordzilla) [inline .github/workflows/lint.yml]: > nitpick

Please add a newline at the end of file.

**Review by nordzilla — CHANGES_REQUESTED** [review]

**Comment by connorkuehl** [comment]
@nordzilla done! PTAL

**Review by nordzilla — APPROVED** [review]
LGTM!

**Pull request merged**